modules/gitops-argo-cd-installation.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
```

Comment:
Not a blocker, but FYI, you might want to update your "module included in" notes for these modules

Reply:
I'd say this should be updated before merging, please.

Reply:
+1, it should be included before it's merged.

Reply:
done